rpp-charter.md

```diff
@@ -3,46 +3,50 @@
-patterns, tools and technologies create a desire to have a provisioning protocol using the [REST](https://ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) 
-architectural style and the [JSON](https://datatracker.ietf.org/doc/html/rfc8259) data-interchange format.
-Such design is expected to take advantage of stateless architecture and widely deployed solutions such as OpenAPI with documentation,
-testing and code generation tools and L4-L7 network services such as API gateways, authorization servers, load balancers, web servers, WAFs etc.
-The successful adoption of RDAP by Domain Name Registries (DNRs) and Regional Internet Registries (RIRs) demonstrates the usefulness of this type of architecture and technology tool stack.
-
-This evolution already started in the ccTLD space and first production deployments of this approach have already seen
-adoption by both existing clients and a preferred way of integration for new registrars.
+patterns, tools and technologies create a desire to have a provisioning protocol using the 
+[REST](https://ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) architectural style and the 
+[JSON](https://datatracker.ietf.org/doc/html/rfc8259) data-interchange format. Such design is expected to 
+take advantage of stateless architecture and widely deployed solutions such as OpenAPI, associated documentation,
+testing and code generation tools, and L4-L7 network services such as API gateways, authorization servers, load balancers, 
+web servers, WAFs etc. The successful adoption of RDAP by Domain Name Registries (DNRs) and Regional Internet Registries (RIRs) 
+demonstrates the usefulness of this type of architecture and technology tool stack.
+
+This evolution has already started with some Country Code Top Level Domains (ccTLDs). First production deployments of this approach have seen
+adoption by both new and existing clients, where a preference has been noted for this approach.
 A REST architecture is expected to allow easier integration between registries and registrars, thus lowering the
 costs for domain registration and new market entrants.
 
 # Objective and Scope
 
 This working group is tasked with creating a series of specifications
-to be known collectively as RESTful Provisioning Protocol (RPP).
+to be known collectively as the RESTful Provisioning Protocol (RPP).
 These specifications will specifically target a REST architecture high on the Richardson Maturity Model
 using HTTPS and JSON. 
 
-The working group will evaluate for possible inclusion of extensibility considerations, allowing
-future use cases beyond the domain name provisioning as well as adding
-or extending operations and data models defined in the core set of protocol specifications.
+In addition to use cases and scenarios for domain name registration, this working group will
+consider features and extension mechanisms useful for other types of Internet registration areas
```

Comment:
"Internet registration areas" -> "registration areas" 

maybe future use cases may also include registration for non-Internet related resources?

Reply:
I think "Internet" is what stays in concern of IETF.
I am more concerned about "will evaluate for possible inclusion of extensibility" changed to "will
consider features and extension mechanisms". The previous one was for me more of obligation to evaluate, now is sounds like obligation to consider them. My preference would be to stay with the previous one or to change from will to may, especially that deliverables read "The working group may consider work for the provisioning of other types of Internet registries".

Reply:
> "Internet registration areas" -> "registration areas"
> 
> maybe future use cases may also include registration for non-Internet related resources?

This is to purposefully limit the scope to the things the IETF concerns itself with. Any charter that is too broad is likely to be rejected.



> I think "Internet" is what stays in concern of IETF. I am more concerned about "will evaluate for possible inclusion of extensibility" changed to "will consider features and extension mechanisms". The previous one was for me more of obligation to evaluate, now is sounds like obligation to consider them. My preference would be to stay with the previous one or to change from will to may, especially that deliverables read "The working group may consider work for the provisioning of other types of Internet registries".

"Will evaluate" does not obligate the working group to produce anything, and neither does the new wording. But both do say the working group will do something. The original wording was clunky IMHO.